pkg/util/kubernetes/apiserver/diagnosis.go

```diff
@@ -12,8 +12,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// DiagnoseName is used as an identifier to Kubernetes API Server connectivity check
+const DiagnoseName = "Kubernetes API Server availability"
```

Comment:
can we remove this public `const` as well?

Reply:
done 👍